### PR TITLE
docs: Add changelog of 4.6.4

### PR DESCRIPTION
**Pull request by dengfuping**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Add changelog of 4.6.4.          |
| 🇨🇳 Chinese | 增加 4.6.4 版本的更新日志。          |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered CHANGELOG.en-US.md](https://github.com/ant-design/ant-design/blob/release-4.6.4/CHANGELOG.en-US.md)
[View rendered CHANGELOG.zh-CN.md](https://github.com/ant-design/ant-design/blob/release-4.6.4/CHANGELOG.zh-CN.md)

### CODE REVIEW

**Review by xrkffgg — APPROVED** [review]

**Review by zombieJ — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -294,15 +294,15 @@
   "bundlesize": [
     {
       "path": "./dist/antd.min.js",
-      "maxSize": "300 kB"
+      "maxSize": "320 kB"
```
Comment (afc163) [inline package.json:297]: 先别改这个，留着 @zombieJ 来修复。

File: package.json [hunk 2]
```diff
@@ -294,15 +294,15 @@
...
     },
     {
       "path": "./dist/antd.min.css",
       "maxSize": "65 kB"
     },
     {
       "path": "./dist/antd.dark.min.css",
-      "maxSize": "66 kB"
+      "maxSize": "67 kB"
```
Comment (afc163) [inline package.json:305]: 同上。

File: package.json (hunk 1 above)
Comment (dengfuping) [reply package.json:297]: - 那等 @zombieJ 修复完，再调整回原先的设置吧，现在主要是为了 CI 能过。应该也不影响今天的发版吧。

**Comment by yoyo837** [comment]
豆酱说有重复包需要改依赖版本一致来解决，改之前发版不合适吧！

**Comment by dengfuping** [comment]
- https://github.com/ant-design/ant-design/commit/8ab88629074b24567965ad9ce7dc54ce93252edf 解决了 rc-dialog 重复依赖的问题，是指这个吗？

**Comment by zombieJ** [comment]
@yoyo837 这个 patch 版本会先实验一下重构后的 `rc-trigger` 效果如何。如果没有 bug，下个 patch 会全部更新上来~

**Comment by yoyo837** [comment]
那release log说明一下比较好，免得吃issue

**Pull request merged**